src/nvim/normal.c

```diff
@@ -3612,6 +3626,108 @@ void scroll_redraw(int up, long count)
 }
 
 /*
+ * Return length of line "lnum" for horizontal scrolling.
+ */
+static colnr_T scroll_line_len(linenr_T lnum)
+{
+    char_u  *p;
+    colnr_T col;
+    int   w;
```

Comment:
Move to where `w` is assigned. This is absolutely not needed in the function scope.